legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/LegendH2Extensions.java

```diff
@@ -309,4 +311,24 @@ else if (value instanceof Integer || value instanceof Long)
             throw new RuntimeException(e);
         }
     }
+
+    public static Value legend_h2_extension_edit_distance(Value string1, Value string2) throws Exception
+    {
+        if (string1 == ValueNull.INSTANCE || string2 == ValueNull.INSTANCE)
+        {
+            throw new IllegalArgumentException("Null values not supported in legend_h2_extension_edit_distance");
```

Comment:
Should return null rather than failing. Same goes for other method

Reply:
Done